test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md

```diff
@@ -1,32 +0,0 @@
----
```

Comment:
Hey @joaedwar if this test case no longer valid for RHMI?

If it's still valid, I would keep that in the repo.

Reply:
We are no longer updating RHMI docs, just maintaining them. This is for RHOAM docs. So, I think this should be deleted.